### PR TITLE
Add creating with DateTime formatter

### PR DESCRIPTION
**Pull request by golem131**
* Add types

* Add creating with DateTime formatter

* Update scalatest 3.0.0 -> 3.0.1

* Update Scala 2.12.0 -> 2.12.1

* Update sbt 0.13.12 -> 0.13.13

### CODE REVIEW

File: src/main/scala/com/markatta/timeforscala/factories.scala [hunk 1]
```diff
@@ -213,3 +218,23 @@ object YearMonth {
   def apply(zone: ZoneId): YearMonth = java.time.YearMonth.now(zone)
 
 }
+
+object DateTimeFormatter {
+  val ISO_LOCAL_DATE: DateTimeFormatter = java.time.format.DateTimeFormatter.ISO_LOCAL_DATE
```
Comment (johanandren) [inline src/main/scala/com/markatta/timeforscala/factories.scala]: Constants with Scala is CamelCased with initial capital, this is Java style.

**Review by johanandren — COMMENTED** [review]
LGTM with the constant naming fixed

File: src/main/scala/com/markatta/timeforscala/factories.scala (hunk 1 above)
Comment (golem131) [reply src/main/scala/com/markatta/timeforscala/factories.scala]: @johanandren fixed

**Comment by golem131** [comment]
@johanandren - updated

**Review by johanandren — APPROVED** [review]
Thanks!

**Pull request merged**